schema.sql

```diff
@@ -68,4 +68,12 @@ CREATE TABLE visits (
 	vet_id INT REFERENCES vets(id),
 	animal_id INT REFERENCES animals(id),
 	date_of_visit DATE NOT NULL
-);
\ No newline at end of file
+);
+
+--------------------------------------------------------
+
+ALTER TABLE animals ADD COLUMN visits_counter INT;
+
+CREATE INDEX vet_index ON visits(vet_id);
```

Comment:
- [ ] it would be better to create the index for `animal_id ` also in order to increase the performance